cognito/handler.js

```diff
@@ -0,0 +1,5 @@
+export const autoConfirmUser = async (event, context) => {
```

Comment:
I suggest this to be named differently. Two reasons why:

1. If we need another function for the Cognito, `handler` is already taken the name
2. The name of the fine does not indicate what is this function doing

I would switch the things around. The file should be called `autoConfirmUser.js` and the function should be `main`. We need to be consistent with the naming, and `main` is used in all functions

Reply:
And if you do not need the `context`, there is no reason to have it in the function

Reply:
good points, Thanks @petarblazevski 
fixed